newIDE/app/src/InstancesEditor/index.js

```diff
@@ -658,6 +695,17 @@ export default class InstancesEditor extends Component<Props> {
     );
   }
 
+  getTileMapTileSelection = () => {
+    return this.props.tileMapTileSelection;
+  };
+
+  getSelectedInstancesObjectFillColor = (
+    isLocked: boolean
+  ): {| color: number, alpha: number |} => {
+    if (this.props.tileMapTileSelection) return { color: 0xfff, alpha: 0 };
+    return { color: isLocked ? 0xbc5753 : 0x6868e8, alpha: 1 };
```

Comment:
I know you just moved them from another file, but maybe worth checking on light theme + moving them to the gdevelop theme

Reply:
These colors are not related with the theme but I can add a comment to explain